### PR TITLE
@alloy => Add a couple of steps to Installation

### PR DESCRIPTION
**Pull request by mzikherman**
A couple steps I needed to do to get this setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ Want to know more about Emission? Read the [mobile](http://artsy.github.io/blog/
    * `$ brew link pcre`
    * `$ brew install watchman --HEAD`
 3. Install NPM modules: `$ yarn install`
-4. Install Pods: `$ cd Example && pod install`
+4. Install Cocoapods: `$ gem install cocoapods`
+5. Install Pods: `$ cd Example && ARTSY_STAFF_MEMBER=true pod install`
```
Comment (mzikherman) [inline README.md]: I was getting a crash on a fonts pod until I added this ENV var (and `pod install` prompted me to login as well).

**Comment by alloy** [comment]
Good stuff.

**Comment by alloy** [comment]
@mzikherman Oops, I missed a missing capital in ‘Cocoapods’. Pushed a fix https://github.com/artsy/emission/commit/946534ed48913ad195daf5b18ba472f3ccd1fca1

**Comment by mzikherman** [comment]
👻

File: README.md (hunk 1 above)
Comment (orta) [inline README.md]: ah yeah, nice one, this is in our mobile team onboarding 👍

**Pull request merged**